src/app/screens/LibraryScreen/components/FlatListItem/index.js

```diff
@@ -0,0 +1,29 @@
+import React, { useEffect } from 'react';
+import { Animated, TouchableOpacity } from 'react-native';
+
+import Book from '@components/Book';
+
+import { RANGE, SUPERPOSITION } from './constants';
+
+function FlatListItem({ onPress, index, item }) {
+  const opacity = new Animated.Value(0);
```

Comment:
Use `useRef` for this.

```suggestion
  const opacity = useRef(new Animated.Value(0)).current;
```